<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Youtube Downloader</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="dl_progress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>371</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="link_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Link:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="link_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>391</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dl_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_res">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>370</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="save_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save to:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_text">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="find_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loc_button">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>370</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Loc</string>
    </property>
   </widget>
   <widget class="QLabel" name="nov_label">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>223</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Views:</string>
    </property>
   </widget>
   <widget class="QLabel" name="durasi_label">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>253</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Durasi Video: </string>
    </property>
   </widget>
   <widget class="QLabel" name="rating_label">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>283</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Peringkat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnail_set">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>145</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pub_label">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>173</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Publish Date:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="judul">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="pub_date">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>193</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="view">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>223</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="durasi">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>253</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="peringkat">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>283</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="author_label">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>143</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Channel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="author">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
